web_src/js/features/comp/TextExpander.ts

```diff
@@ -1,5 +1,41 @@
-import {matchEmoji, matchMention} from '../../utils/match.ts';
+import {matchEmoji, matchMention, matchIssue} from '../../utils/match.ts';
 import {emojiString} from '../emoji.ts';
+import {svg} from '../../svg.ts';
+import {parseIssueHref} from '../../utils.ts';
+import {createElementFromAttrs, createElementFromHTML} from '../../utils/dom.ts';
+import {getIssueColor, getIssueIcon} from '../issue.ts';
+import {debounce} from 'perfect-debounce';
```

Comment:
One more thing, there is already a "debounce", so no need to introduce another one?

![image](https://github.com/user-attachments/assets/0a17768a-fc20-47c2-90c6-c7599e882887)

![image](https://github.com/user-attachments/assets/e415dbdc-e58b-4192-9146-bc2c9ee6cef6)

Reply:
The existing debounce is not async safe.